src/main/java/org/elasticsearch/action/admin/cluster/state/ClusterStateRequest.java

```diff
@@ -127,6 +133,7 @@ public void readFrom(StreamInput in) throws IOException {
         metaData = in.readBoolean();
         blocks = in.readBoolean();
         indices = in.readStringArray();
+        indicesOptions = IndicesOptions.readIndicesOptions(in);
```

Comment:
we need to add a version check here on the 1.x branch, to make sure that we maintain bw compatibility:
```
if (in.getVersion().onOrAfter(Version.1_5_0)) {
    indicesOptions = IndicesOptions.readIndicesOptions(in);
}
```